packages/charts/theme/lumo/vaadin-chart-styles.js

```diff
@@ -0,0 +1,84 @@
+import '../vaadin-chart-base-theme.js';
+import '@vaadin/vaadin-lumo-styles/color.js';
+import '@vaadin/vaadin-lumo-styles/typography.js';
+import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/register-styles.js';
+
+const chartColors = css`
+  :host {
+    --vaadin-charts-color-0: #5ac2f7;
+    --vaadin-charts-color-1: #1676f3;
+    --vaadin-charts-color-2: #ff7d94;
+    --vaadin-charts-color-3: #c5164e;
+    --vaadin-charts-color-4: #15c15d;
+    --vaadin-charts-color-5: #0e8151;
+    --vaadin-charts-color-6: #c18ed2;
+    --vaadin-charts-color-7: #9233b3;
+    --vaadin-charts-color-8: #fda253;
+    --vaadin-charts-color-9: #e24932;
+    --vaadin-charts-color-positive: var(--vaadin-charts-color-4, #15c15d);
+    --vaadin-charts-color-negative: var(--vaadin-charts-color-9, #e24932);
+    --vaadin-charts-background: var(--lumo-base-color);
+    --vaadin-charts-title-label: var(--lumo-header-text-color);
+    --vaadin-charts-axis-title: var(--lumo-secondary-text-color);
+    --vaadin-charts-axis-label: var(--lumo-secondary-text-color);
+    --vaadin-charts-data-label: var(--lumo-body-text-color);
+    --vaadin-charts-secondary-label: var(--lumo-secondary-text-color);
+    --vaadin-charts-axis-line: var(--lumo-contrast-5pct);
+    --vaadin-charts-grid-line: var(--lumo-contrast-20pct);
+    --vaadin-charts-disabled-label: var(--lumo-disabled-text-color);
+    --vaadin-charts-contrast: var(--lumo-contrast);
+    --vaadin-charts-contrast-5pct: var(--lumo-contrast-5pct);
+    --vaadin-charts-contrast-10pct: var(--lumo-contrast-10pct);
+    --vaadin-charts-contrast-20pct: var(--lumo-contrast-20pct);
+    --vaadin-charts-contrast-60pct: var(--lumo-contrast-60pct);
+    --vaadin-charts-tooltip-background: var(--lumo-base-color);
+    --vaadin-charts-tooltip-border-color: inherit;
+    --vaadin-charts-button-label: var(--lumo-primary-text-color);
+    --vaadin-charts-button-background: var(--lumo-contrast-5pct);
+    --vaadin-charts-button-hover-background: var(--lumo-primary-color-10pct);
+    --vaadin-charts-button-active-label: var(--lumo-primary-contrast-color);
+    --vaadin-charts-button-active-background: var(--lumo-primary-color);
```

Comment:
I would suggest to move everything except  --vaadin-charts-color-*` to `chartTheme`.

Reply:
Done.